website/content/labs/lab5.md

```diff
@@ -110,13 +99,17 @@ Create a new file in the `tests/` directory called `test_segment.c`. You’ll ne
 ```
 
 {{% warning %}}
+{{% md %}}
 **Warning**: From now on, you must do all of your work over `ssh`, or in CSIL, or using the [department’s virtual machine](https://howto.cs.uchicago.edu/vm:index). Each of those environments have the `criterion` library properly set up and ready to go. While it is possible to install `criterion` on an unsupported machine, we may not be able to provide support for that setup.
```

Comment:
We should make our first suggestion here the headless VM

Reply:
Fixed